src/components/Modal/Modal.tsx

```diff
@@ -1,89 +1,124 @@
 /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
```

Comment:
nit: what do you think about setting this rule off globally?

Reply:
Already done